client/src/components/Indicators.tsx

```diff
@@ -190,6 +190,8 @@ class IndicatorsWithoutI18n extends Component<IndicatorsProps, IndicatorsState>
 
       const datasetDescription = INDICATORS_DATASETS[this.state.activeVis];
 
+      justfix_startFullStoryRecording();
```

Comment:
This is being called in `render()`, which I don't like, because it's supposed to be a "pure" function that has no side effects.  But when I put it in `componentDidMount()`, it got called even when the user wasn't actually on the timeline tab... Anyways, since this just starts recording and isn't particularly related to the lifecycle of the component, I think it's okay to just leave it in `render()`, but whatevs.

If we want to get really fancy we could make a separate `<StartFullStory/>` component that calls `justfix_startFullStoryRecording()` on mount, but that might be going overboard.